packages/react/src/components/Modal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Modal, ModalSize } from ":/components/Modal/index";
+import { useCunningham } from ":/components/Provider";
+import { Button } from ":/components/Button";
+import { DecisionModalProps } from ":/components/Modal/ModalProvider";
+
+export type ConfirmationModalProps = DecisionModalProps;
+
+export const ConfirmationModal = ({
+  title,
+  children,
+  onDecide,
```

Comment:
I wound `onConfirm` more relevant to keep the semantic of the good old `window.confirm` 😅

Reply:
I agree but the thing is that it is an inherited props from `DecisionModalProps` which is use by multiple pre built modals such as delete confirmation modals, and tomorrow maybe a modal which does not really imply a confirmation but multiple choices, WDYT?

Reply:
Ok!